webrtc-stats.html

```diff
@@ -503,6 +511,84 @@
                   [[!RFC7509]] section 3.1 and Appendix A.b.
                   </p>
                 </dd>
+                <dt>
+                  <dfn><code>burstPacketsLost</code></dfn> of type <span class=
+                  "idlMemberType"><a>unsigned long</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  The cumulative number of RTP packets lost during loss
+                  bursts, Appendix A (c) of [[!RFC6958]].
+                  </p>
+                </dd>
+                <dt>
+                  <dfn><code>burstPacketsDiscarded</code></dfn> of type <span class=
+                  "idlMemberType"><a>unsigned long</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  The cumulative number of RTP packets discarded during
+                  discard bursts, Appendix A (b) of [[!RFC7003]].
+                  </p>
+                </dd>
+                <dt>
+                  <dfn><code>burstLossCount</code></dfn> of type <span class=
+                  "idlMemberType"><a>unsigned long</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  The cumulative number of bursts of lost RTP packets,
+                  Appendix A (e) of [[!RFC6958]]. [[!RFC3611]] recommends a Gmin
+                  (threshold) value of 16 for classifying packet loss or discard burst.
+                  </p>
+                </dd>
+                <dt>
+                  <dfn><code>burstDiscardCount</code></dfn> of type <span class=
+                  "idlMemberType"><a>unsigned long</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  The cumulative number of bursts of discarded RTP packets,
+                  Appendix A (e) of [[!burst-gap-discard]].
+                  </p>
+                </dd>
+                <dt>
+                  <dfn><code>burstLossRate</code></dfn> of type <span class=
+                  "idlMemberType"><a>double</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  The fraction of RTP packets lost during bursts, Appendix A (a) of [[!RFC7004]].
+                  </p>
+                </dd>
+                <dt>
+                  <dfn><code>burstDiscardRate</code></dfn> of type <span class=
+                  "idlMemberType"><a>double</a></span>
+                </dt>
+                <dd>
+                  <p>
+                  The fraction of RTP packets discarded during bursts, Appendix A (e) of [[!RFC7004]].
```

Comment:
@alvestrand the packetsExpected is known by the RTP stack. It is already used in fractionLost and is based on RFC3550. see https://tools.ietf.org/html/rfc3550#appendix-A.3